semanticdb/semanticdb3/semanticdb3.md

```diff
@@ -2101,6 +2101,1264 @@ Synthetics are unspecified in SLS, and our experience [\[38\]][38] shows that
 reverse engineering synthetics is very hard. We may improve
 on this in the future, but this is highly unlikely.
 
+### Java
+
+In this section, we exhaustively map Java language features onto SemanticDB.
+As a reference, we use the Java Language Specification [\[68\]][68]
+(referred to as "JLS" in the text below), as well as the Java Virtual Machine Specification [\[69\]][69] (referred to as "JVMS" in the text below).
+
+<a name="java-language"></a>
+#### Language
+
+<table>
+  <tr>
+    <td><b>Value</b></td>
+    <td><b>Explanation</b></td>
+  </tr>
+  <tr>
+    <td><code>Language("Java")</code></td>
+    <td>Java of unknown or unspecified version.</td>
+  </tr>
+  <tr>
+    <td><code>Language("JavaXY")</code></td>
+    <td>Scala, version <code>X.Y</code>.</td>
+  </tr>
+</table>
+
+<a name="scala-symbol"></a>
```

Comment:
Looks copy/pasted from the Scala part of the spec? I assume, this is work in progress and needn't be reviewed?

Reply:
Done.